### DISCUSSION TITLE
MSSQL Server connections via Windows Authentication

### DISCUSSION THREAD

**Discussion by celestialvigor** [vincentlauvlwj/Ktorm#303] [Q&A] [answered]
Hello everyone,

We are considering using Kotlin and Ktorm in my company's development environment, and one of our questions was regarding Ktorm's support for MSSQL Server connections done via Windows Authentication mode (as opposed to SQL Server Authentication mode).

It'd be great if anyone could enlighten me in that regard, as I couldn't find a clear direction in the docs.

**Comment by vincentlauvlwj** [comment, marked as answer]
Just connect like this:

```kotlin
val database = Database.connect("jdbc:sqlserver://MyServer:integratedSecurity=true;")
```

In production environment, I'll be better to use a connection pool: 

```kotlin
val datasource = ... // Any java.sql.DataSource implementation is OK. 
val database = Database.connect(datasource)
```

**Reply by celestialvigor** [reply]
This worked perfectly.
Thank you so much Vincent.
